modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -993,9 +993,9 @@ class CV_EXPORTS_W Subdiv2D
 
     /** @brief Returns the location of a point within a Delaunay triangulation.
 
-    @param pt – Point to locate.
-    @param edge – Output edge that the point belongs to or is located to the right of it.
-    @param vertex – Optional output vertex the input point coincides with.
+    @param pt - Point to locate.
+    @param edge - Output edge that the point belongs to or is located to the right of it.
+    @param vertex - Optional output vertex the input point coincides with.
```

Comment:
Do you want me to replace those dashes with ":" or spaces or just leave them there?